client/homescreen/activity-panel/test/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import ActivityPanel from '../';
+
+// Mock the panels.
+jest.mock( '../panels', () => {
+	return {
+		getAllPanels: jest.fn().mockImplementation( () => [
+			{
+				title: 'custom-panel-1',
+				count: 10000,
+				initialOpen: true,
+				panel: <span>Custom panel 1</span>,
+			},
+			{
+				title: 'custom-panel-2',
+				count: 20000,
+				initialOpen: false,
+				panel: <span>Custom panel 2</span>,
+			},
+		] ),
+	};
+} );
+
+// Mock the panels.
```

Comment:
This comment should be "mock the orders"?

Reply:
Fixed in the commit a5dd3fc